src/components/Issues/ConformaViolationSummaryCard.tsx

```
@@ -0,0 +1,130 @@
+
+const ConformaViolationSummaryCardContent: React.FC<{ applicationName: string }> = ({
+  applicationName,
+}) => {
+  const namespace = useNamespace();
+  const { allResults, loaded, settling, error } = useApplicationConformaResults(applicationName);
+
+  const policyTabUrl =
+    APPLICATION_DETAILS_PATH.createPath({
+      workspaceName: namespace,
+      applicationName,
+    }) + '/conforma-results';
```

ESLint

[error] Unexpected string concatenation.

(prefer-template)